plugins/pulp_rpm/plugins/profilers/yum.py

```diff
@@ -452,10 +458,16 @@ def _translate_erratum(unit, repo_ids, consumer, conduit):
 
         applicable_rpms, upgrade_details = YumProfiler._rpms_applicable_to_consumer(consumer,
                                                                                     updated_rpms)
-        if applicable_rpms:
```

Comment:
I did.  mainly because I did not see value in conditionally logging only when packages where found.  Seemed fine and more helpful, consistent to log the empty list (or None).